KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml

```diff
@@ -601,6 +604,7 @@
     conditional_formatting_include_nulls: false
     series_types: {}
     defaults_version: 1
+    refresh: '10000'
```

Comment:
What is this refresh rate? 10k seconds? 10 seconds is nice because it lets you know that it has finished updating

Reply:
i don't know how that happened maybe it just changed it to 10k ms == 10 seconds??? ill change it back in any case